config/ingest.py

```diff
@@ -0,0 +1,45 @@
+from lsst.obs.auxTel.ingest import AuxTelParseTask
+
+config.parse.retarget(AuxTelParseTask)
+
+config.parse.translation = {
+    'expTime': 'EXPTIME',
+    'object': 'OBJECT',
+    'imageType': 'IMGTYPE',
+    'testType': 'TESTTYPE',
+    'filter': 'FILTER',
+    'lsstSerial': 'LSST_NUM',
+    'date': 'DATE-OBS',
+    'dateObs': 'DATE-OBS',
+    # Note that we DO ingest a run number, but it comes from the path and not the header,
+    # so an entry here generates a warning. It is picked up in the ingest.py override
+    # of getInfo() and registers correctly like this with the line below left commented out.
+    # 'run': 'RUNNUM'
```

Comment:
Sure, sounds good, I've removed. Just to be clear though, this is not the same thing as a visit number, and I'm not sure that for normal operations that a run number is actually something that we'll want/need.

Reply:
Oh yes, for sure. I'd assume that between `OBSTYPE, IMGTYPE` and `OBJECT` we'd have that covered, and I wanted to remove `RUNNUM` in the meantime as it's confusing as it's not currently being used quite like that in the lab data we have, but it could well end up augmenting the keywords I just mentioned on the sky, and might want to be added back in in the future.